test/unit/date/parse.js

```diff
@@ -269,6 +274,8 @@ QUnit.test( "should parse period (a)", function( assert ) {
 	date2 = startOf( date2, "hour" );
 	assertParse( assert, "5 AM", "h a", cldr, date1 );
 	assertParse( assert, "5 PM", "h a", cldr, date2 );
+	assertParse( assert, "上午5", "ah", zh, date1 );
+	assertParse( assert, "下午5", "ah", zh, date2 );
```

Comment:
Can you explain what this assertion does? What's the purpose of `assertParse`, what are the arguments doing?